src/guide/list.md

```diff
@@ -268,25 +268,24 @@ Similar to template `v-if`, you can also use a `<template>` tag with `v-for` to
 Note that it's **not** recommended to use `v-if` and `v-for` together. Refer to [style guide](../style-guide/#avoid-v-if-with-v-for-essential) for details.
 :::
 
-When they exist on the same node, `v-for` has a higher priority than `v-if`. That means the `v-if` will be run on each iteration of the loop separately. This can be useful when you want to render nodes for only _some_ items, like below:
+When they exist on the same node, `v-if` has a higher priority than `v-for`. That means the `v-if` condition will not have access to variables from the scope of the `v-for`:
```

Comment:
```suggestion
When they exist on the same node, `v-if` has a higher priority than `v-for`. That means the `v-if` condition will not have access to variables from `v-for` scope:
```

Reply:
The wording 'the scope of the `v-for`' was my suggestion.

I don't think ending the sentence with '`v-for` scope' quite works in this context. To my ears, '`v-for` scope' refers to the concept in general whereas we're trying to refer to a specific `v-for` mentioned earlier in the paragraph.

Reply:
Oh, that makes total sense for me 👍